src/lib/utils.js

```diff
@@ -5,3 +5,6 @@ export function makeLogger(prefix) {
 }
 
 export const idForItem = item => `${item.time}-${item.url}`;
+
+export const getLangDir = lang =>
```

Comment:
One last tweak here to further simplify the language direction thing, since I had this code copypasta'd in two places